shimmingtoolbox/cli/mask.py

```diff
@@ -128,3 +129,65 @@ def threshold(fname_input, output, thr):
     nib.save(nii_img, output)
     click.echo(f"The filename for the output mask is: {os.path.abspath(output)}")
     return output
+
+
+@mask_cli.command(context_settings=CONTEXT_SETTINGS,
+                  help="Creates a SCT (SpinalCordToolbox) mask from the input file. Depending on the shape (cylinder,"
```

Comment:
SO these help messages really clutter things up. I have a different way of doing these types of messages, which I'll be implementing in my PR and I'll fix these up, too.   Just helps readability